rsa/randnum.py

```diff
@@ -96,3 +96,15 @@ def randint(maxvalue):
         tries += 1
 
     return value
+
+def randrange(start,end):
+    """Returns a random integer from range(start,end)
+    """
+    assert end>start
+    span=end-start
+    #get an int with at least 64 extra bits
+    #because of the extra bits, value%span wraps around at least 2^64 times
+    #the non-uniformity of the resulting distribution is below 2**-64
```

Comment:
This comment doesn't make much sense unless you already know what's being done here. To me it reads "add extra bits, then handle the trouble we caused by adding extra bits".